packages/gatsby-theme-docs/src/utils/helpers/generic.js

```diff
@@ -0,0 +1,40 @@
+import unified from 'unified';
+import markdown from 'remark-parse';
+import remark2react from 'remark-react';
+
+export const markdown2React = markdownString => {
+  const result = unified()
+    .use(markdown, { commonmark: true })
+    .use(remark2react)
+    .processSync(markdownString).contents;
+
+  return result;
+};
+
+export const computeType = value => {
```

Comment:
Can you explain what does this function do? Where is this supposed to be used?

Reply:
It's basically an efficient JavaScript type checker.

Please read about it here:

https://blog.logrocket.com/javascript-typeof-2511d53a1a62/

Reply:
Thanks for the link. In those cases please always add an explanation as comment next to the function and/or link to some resources where the implementation is take from.

However, I still don't understand where is this supposed to be used 😅